### ISSUE TITLE
Perform a "where ... in ..." passing a list of possible matches

### ISSUE THREAD

**Issue by churcho** [mathieuprog/query_builder#1]
I would like to run a query that runs `where(q, [u], u.status in ^status)`
My status here would be a list of status say ["APPROVED", "SUSPENDED"]

None of the available where params seem to have the ability to have pass that structure, seems like the only possible way around is is to Enum and add `or_where` clauses.

**Comment by mathieuprog** [comment]
I added the operators `:in`, `:not_in`, `:include` and `:exclude` in version 0.13.1
```
User
|> QueryBuilder.where({:name, :in, ["Alice", "Bob"]})
```
```
User
|> QueryBuilder.where({:name, :not_in, ["Alice", "Bob"]})
```
```
Article
|> QueryBuilder.where({:tags, :include, "foo"})
```
```
Article
|> QueryBuilder.where({:tags, :exclude, "foo"})
```

**Comment by mathieuprog** [comment]
I pushed too fast and 0.13.0 contains a bug that has been fixed in 0.13.1. Make sure to depend on 0.13.1.
